src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -433,11 +434,38 @@ def test_fir_filter_coefficients():
 '''
 
 
+def test_send_software_edge_trigger_start_old(session):
```

Comment:
I feel the name - `test_send_software_edge_trigger_start_deprecated` would be more descriptive.

Reply:
Changed